pages/feed/index.tsx

```diff
@@ -294,89 +298,91 @@ const Feeds: WithPageLayout<HighlightSSRProps> = (props: HighlightSSRProps) => {
             </DialogContent>
           </Dialog>
         )}
-        <Tabs onValueChange={onTabChange} defaultValue="home" className="w-full 2xl:max-w-[40rem] xl:max-w-[33rem]">
-          <TabsList className={clsx("justify-start  w-full border-b", !user && "hidden")}>
-            <TabsTrigger
-              className="data-[state=active]:border-sauced-orange data-[state=active]:border-b-2 text-lg"
-              value="home"
-            >
-              Home
-            </TabsTrigger>
-            <TabsTrigger
-              className="data-[state=active]:border-sauced-orange  data-[state=active]:border-b-2 text-lg"
-              value="following"
-            >
-              Following
-            </TabsTrigger>
-          </TabsList>
-
-          {user && (
-            <div className="flex max-w-3xl px-1 pt-4 lg:gap-x-3">
-              <div className="hidden lg:inline-flex pt-[0.4rem]">
-                <Avatar
-                  alt="user profile avatar"
-                  isCircle
-                  size="sm"
-                  avatarURL={`https://www.github.com/${user?.user_metadata.user_name}.png?size=300`}
-                />
-              </div>
+        <div className="flex flex-col w-full gap-12 md:items-start lg:flex-row">
+          <Tabs onValueChange={onTabChange} defaultValue="home" className="w-full 2xl:max-w-[40rem] xl:max-w-[33rem]">
+            <TabsList className={clsx("justify-start  w-full border-b", !user && "hidden")}>
+              <TabsTrigger
+                className="data-[state=active]:border-sauced-orange data-[state=active]:border-b-2 text-lg"
+                value="home"
+              >
+                Home
+              </TabsTrigger>
+              <TabsTrigger
+                className="data-[state=active]:border-sauced-orange  data-[state=active]:border-b-2 text-lg"
+                value="following"
+              >
+                Following
+              </TabsTrigger>
+            </TabsList>
 
-              <HighlightInputForm refreshCallback={mutate} />
-            </div>
-          )}
-          <TabsContent value="home">
-            <HomeHighlightsWrapper emojis={emojis} mutate={mutate} highlights={data} loading={isLoading} />
-            {meta.pageCount > 1 && (
-              <div className="flex items-center justify-between max-w-3xl px-2 mt-10">
-                <div className="flex items-center w-max gap-x-4">
-                  <PaginationResults metaInfo={meta} total={meta.itemCount} entity={"highlights"} />
+            {user && (
+              <div className="flex max-w-3xl px-1 pt-4 lg:gap-x-3">
+                <div className="hidden lg:inline-flex pt-[0.4rem]">
```

Comment:
maybe you can remove `0.4rem` and tried with `pt-1.5` -> '0.375rem'?

Reply:
That was already there when I started so I didn't touch it.